app/views/users/new.html.erb

```diff
@@ -3,4 +3,8 @@
 <%= render 'form' %>
 
 <br>
-<%= link_to fa_icon('chevron-left', text: 'Back'), users_path, class: 'btn btn-sm btn-default' %>
+<br>
```

Comment:
Lets try not to use `<br>` tags, instead us css to add the spacing you are looking for.